content/zh-cn/docs/contribute/style/content-guide.md

```diff
@@ -84,20 +84,18 @@ Docs can link to third-party open source software (OSS) outside the Kubernetes p
 ### 第三方内容 {#third-party-content}
 
 Kubernetes 文档包含 Kubernetes 项目下的多个项目的应用示例。
-这里的 Kubernetes 项目指的是 [kubernetes](https://github.com/kubernetes) 和
-[kubernetes-sigs](https://github.com/kubernetes-sigs) GitHub 组织
-下的项目。
+这里的 Kubernetes 项目指的是 [Kubernetes](https://github.com/kubernetes) 和
+[Kubernetes SIGs](https://github.com/kubernetes-sigs) GitHub 组织下的那些项目。
```

Comment:
Note: The English text is still `kubernetes-sigs`, not `Kubernetes SIGs`.

But I think it's a good change for Mandarin.

Reply:
Yes, it might be better to keep the text same as the linked title here.